DuplicateAssemblyScanner/DuplicateAssemblyScanner/Settings.cs

```diff
@@ -0,0 +1,124 @@
+namespace DuplicateAssemblyScanner {
+    using ColossalFramework.UI;
+    using DuplicateAssemblyScanner.Structs;
+    using DuplicateAssemblyScanner.Util;
+    using ICities;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Generates the settings screen based on cached results from the assembly scanner.
+    /// </summary>
+    public class Settings {
+        // Markers used in checkbox labels
+        private const string MARKER_BLANK = " ";
```

Comment:
The .NET naming convention suggests using `PascalCase` for constants.

Reply:
Will fix as part of #19 for v1.2.1